### PR TITLE
tests: Add docker integration test for attach, commit, cp and create.

### LINKED ISSUE #38: Add docker integration tests for attach, cp, create and commit
These will test docker attach, docker cp, docker create and docker commit functionality.

### PR DESCRIPTION
**Pull request by GabyCT**
These commits will test docker attach, docker commit, docker cp and
docker create functionality.

Fixes #38

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
lgtm

**Comment by chavafg** [comment]
lgtm

File: integration/docker/attach_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		id                string
+		exitCode          int
+		containerExitCode int
+	)
+
+	BeforeEach(func() {
+		containerExitCode = 13
+		id = randomDockerName()
+		_, _, exitCode = dockerRun("--name", id, "-d", Image, "sh", "-c",
+			fmt.Sprintf("sleep 3 && exit %d", containerExitCode))
```
Comment (jcvenegas) [inline integration/docker/attach_test.go]: can we make  the tieme sleep a constant `destroyTimeout`  ?

**Comment by GabyCT** [comment]
@jcvenegas changes applied

**Pull request merged**